src/physics/base_solver.hpp

```diff
@@ -204,6 +204,11 @@ class BaseSolver {
    */
   int order_;
 
+  /**
+   * @brief ParaView data collection pointer
+   */
+  std::unique_ptr<mfem::ParaViewDataCollection> pv_dc_;
```

Comment:
Instead of having separate variables for different types of data collection, it might be preferable to have a 
`std::unique_ptr<mfem::DataCollection>`.

I have something similar implemented here: https://github.com/LLNL/serac/pull/88/files (branch is waiting on an Axom PR)